### DISCUSSION TITLE
Long Wait for GitHub Sponsors Approval is Demotivating 😔

### DISCUSSION THREAD

**Discussion by implabinash** [community/community#140685] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I applied for GitHub Sponsors on **September 15**, and it’s been over three weeks with no updates.

My profile says "We need more information to continue," but I haven’t received any guidance or details. I also submitted a support ticket but was only told that applications are reviewed in order, and nothing has changed since.

This long wait with no clear communication is really demotivating.

I’m excited to keep developing and maintaining my open-source projects, but without a clear path to get sponsorship set up, it’s becoming frustrating. It’s hard to stay motivated when there’s no clarity on when or how the approval will happen.

I hope GitHub can improve this process to help open-source developers move forward faster.

Any feedback or updates would be appreciated!

**Comment by bfox200925** [comment]
Here’s a suggestion for improving your communication with GitHub and increasing the chances of getting a resolution:

**Follow-Up on Your Support Ticket:**
 If it’s been a while since your last contact, consider replying to your initial support ticket to check in again. Sometimes a gentle nudge can help move things along.

**Engage with GitHub Community or Social Media:** 
GitHub staff are often active on platforms like Twitter or the GitHub Community Discussions forum. Bringing visibility to your situation on those platforms (while being respectful) might encourage a faster response.

**Provide Additional Information:** 
Since your profile indicates that more information is needed, it’s worth checking if there's any way for you to proactively offer more details or clarify anything that might be missing.

**Keep Developing Your Projects:**
 While you're waiting, continuing to build and engage with your community around your projects could not only keep your motivation up but also demonstrate to GitHub that you are actively contributing.

I hope you get some clarity and a resolution soon! Best of luck!

**Reply by implabinash** [reply]
Luckily, My application has been approved. Thanks to the GitHub Team.🥳

**Reply by unpeturbed** [reply]
Good to read this. Congratulations!
